clojure/coast/deps.edn

```diff
@@ -6,6 +6,6 @@
  :aliases {:outdated {:extra-deps {olical/depot {:mvn/version "2.0.1"}}
                       :main-opts ["-m" "depot.outdated.main"
                                   "-a" "outdated"]}
-           :uberjar {:extra-deps {uberdeps/uberdeps {:mvn/version "1.0.2"}}
```

Comment:
@malcolmsparks Using fixed version dependencies is hard to maintain. Is there any way in `clojure` to be more relaxed on version strategy ? (Using ranges or else)

Reply:
I would love to know this, too. Every example I have ever seen uses exact versions.

Reply:
From what I can tell from discussions and samples, Clojure requires exact version numbers.

Reply:
ok